cpp/ql/test/library-tests/extraction_errors/PrintAst.qlref

```diff
@@ -0,0 +1 @@
+semmle/code/cpp/PrintAST.ql
```

Comment:
Could be omit the AST and just have tests that output the error expressions and error types with the location where they occur?

Reply:
I think it's actually quite useful to see where the error expressions are in relation to the rest of the AST, so I'd prefer to keep this.

Reply:
My worry is mostly that this will be one more place we need to touch when we change something in PrintAST.